pset-iterators.js

```diff
@@ -137,7 +148,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter((alive) => {
```

Comment:
@deedeemedina nice algorithm to filter out only the living presidents 

Your iterator variable `alive` is fine but makes it harder to read your code. You should instead follow the best practice of naming your iterator the singular version of your array ("presidents") - so naming your iterator `president` instead of `alive` helps make your code easier to follow 

Also, the indentation on line 152 is off, looks like you have indented more than the recommended approach of just 1 level